misc/userscripts/rss

```diff
@@ -0,0 +1,122 @@
+#!/bin/sh
```

Comment:
I run it on Debian, which uses dash as `/bin/sh`, and I've tested it with busybox sh in the past. They barely have any non-POSIX features and I don't use any of them as far as I'm aware.